Changelog.md

```diff
@@ -9,6 +9,8 @@ Features:
 Bugfixes:
  * Disallow unknown options in `solc`
  * Inline assembly: support the `address` opcode
+ * Inline assembly: magic variables (`this`, `super`, etc.) are not visible
+   in inline assembly and they can be used as local variables names
```

Comment:
Everything that is not a keyword can be used as local variable names, they just shadow.

I think the actual fix here is that external variables of unsupported types are properly detected.